docs/source/using-xgi.rst

```diff
@@ -10,20 +10,40 @@ Published work
 2023
 ----
 
-- Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, `"Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes" <http://doi.org/10.1038/s41467-023-37190-9>`_, Nature Communications **14**, 1605 (2023).
-- Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, `"Community detection in large hypergraphs" <https://doi.org/10.1126/sciadv.adg9159>`_, Science Advances **9**, eadg9159 (2023).
+Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, "Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes", *Nature Communications* **14**, 1605 (2023).
+
+:bdg-link-primary-line:`Paper <http://doi.org/10.1038/s41467-023-37190-9>`
+:bdg-link-primary-line:`Github <https://github.com/maximelucas/HOI_shape_sync_differently>`
+
+Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, "Community detection in large hypergraphs", *Science Advances* **9**, eadg9159 (2023).
```

Comment:
I might be doing the greatest nitpick in xgi's history, the rest looks very cool, thank you @nwlandry  

```suggestion
Nicolò Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, "Community detection in large hypergraphs", *Science Advances* **9**, eadg9159 (2023).
```

Reply:
No, that is actually super helpful, thanks so much!! I must have made an error when transcribing :)